logfire/_internal/exporters/wrapper.py

```diff
@@ -36,10 +35,11 @@ class WrapperMetricExporter(MetricExporter):
     def __init__(
         self,
         exporter: MetricExporter,
-        preferred_temporality: dict[type, AggregationTemporality] | None = None,
-        preferred_aggregation: dict[type, Aggregation] | None = None,
     ) -> None:
-        super().__init__(preferred_temporality=preferred_temporality, preferred_aggregation=preferred_aggregation)  # type: ignore
+        super().__init__(  # type: ignore
+            preferred_temporality=exporter._preferred_temporality,
```

Comment:
Reading protected attributes doesn't really feel better

Reply:
Fair enough, I'll just update comments as part of #1048